From 653abbb9299c25fc8bbc185a561e70ef396b9e53 Mon Sep 17 00:00:00 2001
From: David von Oheimb <dev@ddvo.net>
Date: Thu, 11 Aug 2022 17:46:11 +0200
Subject: [PATCH 163/209] 80-test_ssl_new.t: make dependencies on CTLOG_FILE
 and TEST_CERTS_DIR explicit

Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: Shane Lontis <shane.lontis@oracle.com>
(Merged from https://github.com/openssl/openssl/pull/18918)

(cherry picked from commit 8b3ccf1cd590c3f9f63d4665106fa4defbffa51f)
---
 test/recipes/80-test_ssl_new.t | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/test/recipes/80-test_ssl_new.t b/test/recipes/80-test_ssl_new.t
index a1828094db..a8c2429538 100644
--- a/test/recipes/80-test_ssl_new.t
+++ b/test/recipes/80-test_ssl_new.t
@@ -6,6 +6,10 @@
 # in the file LICENSE in the source distribution or at
 # https://www.openssl.org/source/license.html
 
+# For manually running these tests, set specific environment variables like this:
+# CTLOG_FILE=test/ct/log_list.cnf
+# TEST_CERTS_DIR=test/certs
+# For details on the environment variables needed, see test/README.ssltest.md
 
 use strict;
 use warnings;
@@ -163,13 +167,14 @@ sub test_conf {
       skip "No tests available; skipping tests", 1 if $skip;
       skip "Stale sources; skipping tests", 1 if !$run_test;
 
+      my $msg = "running CTLOG_FILE=test/ct/log_list.cnf". # $ENV{CTLOG_FILE}.
+          " TEST_CERTS_DIR=test/certs". # $ENV{TEST_CERTS_DIR}.
+          " test/ssl_test test/ssl-tests/$conf $provider";
       if ($provider eq "fips") {
           ok(run(test(["ssl_test", $output_file, $provider,
-                       srctop_file("test", "fips-and-base.cnf")])),
-             "running ssl_test $conf");
+                       srctop_file("test", "fips-and-base.cnf")])), $msg);
       } else {
-          ok(run(test(["ssl_test", $output_file, $provider])),
-             "running ssl_test $conf");
+          ok(run(test(["ssl_test", $output_file, $provider])), $msg);
       }
     }
 }
-- 
2.34.1

